src/translations/data.json

```
@@ -2383,5 +2429,20 @@
     "sr": "Компактан приказ резултата претраге",
     "mk": "Компактен преглед на резултатите од пребарувањето",
     "bg": "Компактен изглед на резултатите от търсенето"
+  },
+  "orderOfCases": {
+    "en": "Order of cases",
+    "isv": "Poredok padežev",
+    "ru": "Порядок падежей",
+    "uk": "Порядок відмінків",
+    "be": "Парадак склонаў",
+    "pl": "Kolejność przypadków",
+    "cs": "Pořadí případu",
+    "sk": "Poradie prípadov",
```

```suggestion
    "sk": "Poradie pádov",
```
and also in czech most likely "Pořadí pádů"

---

Czech phrase is also changed, thanks